lib/src/base/theme/design_system/design_system.dart

```diff
@@ -3,27 +3,27 @@ import 'design_system_icons.dart';
 import 'design_system_spacings.dart';
 import 'design_system_typography.dart';
 
-class PrimeDesignSystem {
-  const PrimeDesignSystem({
+class DesignSystem {
```

Comment:
Please confirm we do not expose the `DesignSystem` outside of the package. If we do, it will need a prefix. For example `WidgetToolkitDesignSystem`

Reply:
We Expose it, because we use the same DesignSystem in all other packages. Will add the prefix.